run_playlist_generator.py

```diff
@@ -0,0 +1,116 @@
+import os
+import sys
+sys.path.append('./2_image_caption_to_mood/')
+sys.path.append('./3_mood_to_playlist/')
+from transformer import MultiModalEarlyFusionBertModel, idx_to_emotion
+from run_songs_to_playlist import extract_random_songs, mood_mapping, create_spotify_playlist
+
+import requests
+import torch
+from PIL import Image
+from tqdm import tqdm
+from torchvision import transforms
+from torchvision.models import resnet18
+from transformers import AutoModel, AutoConfig, AutoTokenizer, BertConfig
+from transformers import VisionEncoderDecoderModel, GPT2TokenizerFast, ViTImageProcessor
+
+# set device to GPU if available
+device = "cuda" if torch.cuda.is_available() else "cpu"
+
+caption_model = VisionEncoderDecoderModel.from_pretrained("nlpconnect/vit-gpt2-image-captioning").to(device)
+caption_tokenizer = GPT2TokenizerFast.from_pretrained("nlpconnect/vit-gpt2-image-captioning")
+caption_image_processor = ViTImageProcessor.from_pretrained("nlpconnect/vit-gpt2-image-captioning")
+
+def get_caption(image_path):
```

Comment:
I would add a little comment just for documentation to say that we use a different pretrained model from the one we built for accuracy purposes in having the end to end framework?